src/lib/connection/connector/fallback-handlers.js

```diff
@@ -163,3 +163,17 @@ export const getPreferredContentLocale = (content) => {
     return keys[index] || keys[0];
   }
 };
+
+/**
+ * Get the current user's content locale, fallback to branch default locale
+ *
+ * @return {String} the user's content locale
+ */
+export const getUserContentLocale = () => {
+  log.debug('fallback/getUserContentLocale');
+  const locale = getBranchDefaultLanguage().locale;
+
+  // TO-DO: implement user language detection
```

Comment:
@Ninerian here I'm unsure what exactly would be missing / how I can achieve this.